tests/androidresource/src/AndroidAliasesTest.java

```diff
@@ -0,0 +1,32 @@
+import androidx.annotations.FractionRes;
+import androidx.annotations.StringRes;
+import android.annotations.AnyRes;
+import android.annotations.RawRes;
+import android.support.annotation.AttrRes;
+import android.support.annotation.XmlRes;
+
+public class AndroidAliasesTest {
+
+    @AnyRes int any_res;
+    @RawRes int raw_res;
+
+    @FractionRes int fraction_res;

```

Comment:
Instead of 5 different XXXRes annotations, I would just use two - no point in adding all the other files just for this test.

Reply:
Fixed.